src/services/userTransform.ts

```diff
@@ -231,6 +231,12 @@ export class UserTransformService {
       response.body = {
         error: extractStackTraceUptoLastSubstringMatch(error.stack, SUPPORTED_FUNC_NAMES),
       };
+      const metaTags = getTransformationMetadata(events[0]?.metadata);
+
+      stats.counter('user_transform_test_errors', events.length, {
+        status: error.statusCode,
+        ...metaTags,
+      });
```

Comment:
Why not write this in finally block and capture every status code, we can always filter on statuscode != 200 but atleast we will know how many calls to test transform by what workspaceId are being sent. Wdyt ?!

Reply:
we can name the stat `user_transform_test_count_total`

Reply:
Main concern right now is errors. Having total count also makes sense, it might be useful in the future

Reply:
Sounds good, making that change